optimum/habana/transformers/models/gptj/modeling_gptj.py

```diff
@@ -134,6 +125,8 @@ def forward(
         use_cache: Optional[bool] = False,
         output_attentions: Optional[bool] = False,
         token_idx: Optional[torch.Tensor] = None,
+        sin: Optional[torch.Tensor] = None,
```

Comment:
add the description of you changes  follow [line142](https://github.com/huggingface/optimum-habana/pull/468/files#diff-d4a4f3a035d8b9d48e0cb7e517d82c20d384aceaef62fb9f51bb9a355347a1b4L142)

Reply:
added